### PR TITLE
Security context constraints

### PR DESCRIPTION
**Pull request by pweil-**
In preparation for https://github.com/openshift/origin/pull/2609

/cc @pmorie @liggitt

### CODE REVIEW

File: architecture/additional_concepts/authorization.adoc [hunk 1]
```diff
@@ -194,3 +194,60 @@ If you find that these roles do not suit you, a *cluster-admin* user can create
...
+into the set of allowable values.
+
+|===
+
+=== Admission
+
+Admission control with security context constraints allows the ability to control the creation
+of resources based on capabilities granted to a user. In terms of the security context
+constraints it means that an admission controller may inspect the user info made available
+in the context to retrieve and appropriate set of security context constraints to ensure the pod
```
Comment (adellape) [inline architecture/additional_concepts/authorization.adoc]: s/and/an/

**Comment by adellape** [comment]
@pweil- Is https://github.com/openshift/origin/pull/2856 what I should be watching for now?

**Comment by pweil-** [comment]
@adellape - sorry, should have linked.  https://github.com/openshift/origin/pull/2868 is the one that would make the full feature with admission so probably best to watch that one.

File: architecture/additional_concepts/authorization.adoc [hunk 2]
```diff
@@ -194,3 +194,60 @@ If you find that these roles do not suit you, a *cluster-admin* user can create
...
+in the context to retrieve an appropriate set of security context constraints to ensure the pod
+is authorized to make requests about its operating environment or to generate a set of constraints
+to apply to the pod.
+
+The set of security context constraints that admission uses to authorize a pod are determined
+by the user identity and groups that the user belongs to.  Additionally, if the pod specifies
+a service account, the set of allowable security context constraints includes any constraints
+accessible to the serivice account.
+
+Admission ensures uses the following approach to create the final security context for the pod:
```
Comment (adellape) [inline architecture/additional_concepts/authorization.adoc]: s/ensures//

**Comment by adellape** [comment]
@pweil- I see the code has merged.

1) Is the content as submitted here still accurate?

2) Does this affect/warrant any update to the "Security Warning" here? http://docs.openshift.org/latest/admin_guide/install/setup.html#system-requirements

**Comment by pweil-** [comment]
@adellape - content is still good.  I have an example walk through of escalating privileges for a user I want to add in.  

We should probably add in a link to this section in the security warning area since this is how folks can restrict users from running with escalated privileges within their cluster.

**Comment by adellape** [comment]
@pweil- OK thanks. Merging as-is, and will follow-up now w/ any minor edits and a link from the "Security Warning".

**Pull request merged**